server/request_object.cc

```diff
@@ -216,7 +220,7 @@ void RequestObject::initialise() {
 }
 
 RequestObject::~RequestObject() {
-  s3_log(S3_LOG_DEBUG, request_id, "Destructor\n");
+  s3_log(S3_LOG_DEBUG, request_id, "%s Dtor\n", __func__);
```

Comment:
why to log "Dtor" !! __func__ would log the name of the function, and would be understood from this only that this is destructor !!

Reply:
Done.